tendrl/node_agent/monitoring/collectd/collectors/gluster/heavy_weight/tendrl_glusterfs_profile_info.py

```diff
@@ -277,44 +277,111 @@ def process_volume_profile_info(self, volume):
+                self.profile_info[
+                    t_name % (
+                        self.CONFIG['integration_id'],
+                        volName,
+                        brick_host.replace('.', '_'),
+                        brickName.split(':')[1].replace('/', '|')
+                    )
+                ] = (total_reads / int(brick_det.get('intervalStats').get(
+                        'duration'
+                    ))
                 )
-            ] = brick_det.get('intervalStats').get('totalRead')
-            t_name = "clusters.%s.volumes.%s.nodes.%s.bricks.%s.iops." \
-                "gauge-write"
-            self.profile_info[
-                t_name % (
-                    self.CONFIG['integration_id'],
-                    volName,
-                    brick_host.replace('.', '_'),
-                    brickName.split(':')[1].replace('/', '|')
+                t_name = "clusters.%s.volumes.%s.nodes.%s.bricks.%s.iops." \
+                    "gauge-write"
+                self.profile_info[
+                    t_name % (
+                        self.CONFIG['integration_id'],
+                        volName,
+                        brick_host.replace('.', '_'),
+                        brickName.split(':')[1].replace('/', '|')
+                    )
+                ] = (total_writes / int(brick_det.get('intervalStats').get(
+                        'duration'
+                    ))
                 )
-            ] = brick_det.get('intervalStats').get('totalWrite')
-            t_name = "clusters.%s.nodes.%s.bricks.%s.iops." \
-                "gauge-read"
-            self.profile_info[
-                t_name % (
-                    self.CONFIG['integration_id'],
-                    brick_host.replace('.', '_'),
-                    brickName.split(':')[1].replace('/', '|')
+                t_name = "clusters.%s.nodes.%s.bricks.%s.iops." \
+                    "gauge-read"
+                self.profile_info[
+                    t_name % (
+                        self.CONFIG['integration_id'],
+                        brick_host.replace('.', '_'),
+                        brickName.split(':')[1].replace('/', '|')
+                    )
+                ] = (total_reads / int(brick_det.get('intervalStats').get(
+                        'duration'
+                    ))
                 )
-            ] = brick_det.get('intervalStats').get('totalRead')
-            t_name = "clusters.%s.nodes.%s.bricks.%s.iops." \
-                "gauge-write"
-            self.profile_info[
-                t_name % (
-                    self.CONFIG['integration_id'],
-                    brick_host.replace('.', '_'),
-                    brickName.split(':')[1].replace('/', '|')
+                t_name = "clusters.%s.nodes.%s.bricks.%s.iops." \
+                    "gauge-write"
+                self.profile_info[
+                    t_name % (
+                        self.CONFIG['integration_id'],
```

Comment:
@cloudbehl @GowthamShanmugam  do we need to consider Cluster.short_name writing the metric

Reply:
No, not required, we can use integration_id to push the values, we are using cluster_name only for display purpose in grafana and graphite. 

we always better to use /clusters/{integration_id} because during import flow first we are starting collectd and after that only we are creating an alias.